src/locale/locales/fr/messages.po

```diff
@@ -841,43 +822,31 @@ msgstr "Appel soumis"
 msgid "Appeal this decision"
 msgstr "Faire appel de cette décision"
 
-#: src/Navigation.tsx:329
-#: src/screens/Settings/AppearanceSettings.tsx:76
-#: src/screens/Settings/Settings.tsx:175
-#: src/screens/Settings/Settings.tsx:178
+#: src/Navigation.tsx:330
+#: src/screens/Settings/AppearanceSettings.tsx:86
+#: src/screens/Settings/Settings.tsx:183
+#: src/screens/Settings/Settings.tsx:186
 msgid "Appearance"
 msgstr "Affichage"
 
-#: src/view/screens/Settings/index.tsx:476
-#~ msgid "Appearance settings"
-#~ msgstr "Paramètres d’affichage"
-
-#: src/Navigation.tsx:325
-#~ msgid "Appearance Settings"
-#~ msgstr "Paramètres d’affichage"
-
 #: src/screens/Feeds/NoSavedFeedsOfAnyType.tsx:47
 #: src/screens/Home/NoFeedsPinned.tsx:93
 msgid "Apply default recommended feeds"
 msgstr "Utiliser les fils d’actu recommandés par défaut"
 
 #: src/view/com/post-thread/PostThreadItem.tsx:835
 msgid "Archived from {0}"
-msgstr "Archivé par {0}"
+msgstr "Archivé comme du {0}"
```

Comment:
this feels weird in canadian french. from the source code i think this is posts that are imported from another website? maybe using "**Archivé sur**" would be better, but it wouldn't be that big a deal since "Archivé" followed by a website + date should make it obvious to users what this information is

Reply:
I understand your confusion here — but no, here `{0}` is a **date**:

![image](https://github.com/user-attachments/assets/16989188-8409-4645-8baf-c2c858096609)

I wrote it like `Archivé comme du 15 avril 1975`, as if `Archivé comme provenant du 15 avril 1975`. It's not _the best_, but clearly it's better than _Archivé par_ or _Archivé sur_, which would not work with a date.

Do you have any better idea, now that it's clearer `{0}` is filled with a date?

Reply:
"_**Archivé le** 4 avril 1975 à 13:00_" sounds better
```
msgstr "Archivé le {0}"
```
admittedly though i'm not sure why "Archived from" is the original string, i don't think i've seen or heard "Archived from April 4th 1975"

Reply:
It sounds better, but it would be a mistranslation in my opinion: it's not `Archived at` (as in, _this post was archived at that date_), but `Archived from`: _this post was archived as coming from that date_.

I think that's why `from` was used, and that's why I didn't simply chose `Archivé le`, which would have been the easiest choice: it doesn't mean the same thing!

Reply:
We could sidestep the problem by writing it like this:

```
Archive datée du {0}
```

It's a bit longer, but way clearer. What do you think?

Reply:
i like that way better, and it seems like it'd fit the UI, go for it